### PR TITLE
User feedback migrated from pyingest

### PR DESCRIPTION
**Pull request by mugdhapolimera**

### CODE REVIEW

File: adsingestp/parsers/adsfeedback.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+import logging
+
+from adsingestp import utils
+from adsingestp.parsers.base import BaseBeautifulSoupParser
+
+logger = logging.getLogger(__name__)
+
+
+class ADSFeedbackParser(BaseBeautifulSoupParser):
+    def __init__(self, json_string):
```
Comment (seasidesparrow) [inline adsingestp/parsers/adsfeedback.py:11]: I think we have moved toward not requiring that data be loaded in __init__.  Some of the parsers have that option, but I think our standard is to load when the .parse() method is called.  So two things here: one, give the json_string a default value of None so that you don't have to include it, and only initialize it if it's not None; two, add an argument to .parse() where you can pass json_string.

File: adsingestp/parsers/adsfeedback.py [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+        self.data = json.loads(json_string)
+
+    def parse(self, **kwargs):
```
Comment (seasidesparrow) [inline adsingestp/parsers/adsfeedback.py:14]: Again, to be clear, something like `def parse(self, data=None, **kwargs)` followed by `if data: self.data = data`

**Comment by seasidesparrow** [comment]
This looks good, just a minor change to the .init() and .parse() methods so that they can both load data (and default to None, so that it's optional for both), and add the test case with unicode we discussed in slack.

**Comment by seasidesparrow** [comment]
Looks good!  Feel free to merge at your convenience.
